build-deb

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+#
+# Debian/Ubuntu Packaging Scripts
+# Copyright (C) 2002-2021 by Thomas Dreibholz
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
This license seems to conflict with the compiler's license.

Reply:
I assume that this license applies only to the build-deb script, not the compiler itself.

Reply:
Fixed. It is Apache 2.0 now.